src/teaching-element/Video.vue

```diff
@@ -61,7 +61,7 @@ export default {
       };
     },
     style() {
-      return `background-image: url('${this.posterUrl}');`;
+      return `background-image: url('${this.thumbnailUrl}');`;
```

Comment:
Name `poster` is actually more accurate since the thumbnail is a frame from the video and are usually used to preview the video while this is a poster (placeholder) image for before video is played and can be any image.